src/core/issues-notes.js

```diff
@@ -203,6 +193,36 @@ function createIssueSummaryEntry(l10nIssue, report, id) {
   `;
 }
 
+/**
+ *
+ * @param {*} issueList
+ */
+function makeIssueSectionSummary(issueList) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+
+  if (issueSummaryElement) {
+    if (document.querySelectorAll(".issue").length) {
+      const heading = issueSummaryElement.querySelector("h2");
```

Comment:
Note that  `issue-summary` can appear anywhere, and hence there is no guarantee that the first heading will be a h2. It's perfectly valid for it to be any heading level 2-6.